README.md

```diff
@@ -0,0 +1,38 @@
+## Installation
+
+Use the Nuget package manager [SmartShareClient](https://www.nuget.org/packages/SmartShareClient) to install.
+
+```bash
+dotnet add package SmartShareClient
+```
+
+## Usage
+
+```csharp
+services.AddSmartShare(options => 
+{
+	options.Endpoint = "";
+	options.ClientId = "";
+	options.ClientKey = "";
+	options.User = "";
+	options.Password = "";
+});
+```
+
+```csharp
+private readonly ISmartShare smartShareClient;
+
+public Test(ISmartShare smartShareClient)
+{
+	this.smartShareClient = smartShareClient;
+}
+
+public async Task Foo()
+{
+	this.smartShareClient.UploadDocumentAsync(...);
+}
+```
+
+## License
+[MIT](https://choosealicense.com/licenses/mit/)loadDocumentAsync(...);
```

Comment:
There is a layout break here.
![image](https://user-images.githubusercontent.com/70019452/180613016-43a49616-ed01-4177-9ca4-899460fc7633.png)